src/apify/apify_storage_client/_request_queue_client.py

```diff
@@ -157,8 +158,11 @@ async def batch_add_requests(
         *,
         forefront: bool = False,
     ) -> BatchRequestsOperationResponse:
-        return BatchRequestsOperationResponse.model_validate(
-            await self._client.batch_add_requests(
+        processed = []
+        unprocessed = []
+
+        for chunk in chunked(requests, 25):
```

Comment:
I would prefer a constant or a comment explaining why 25, but it's fine since it is just tmp change, thanks